patterns/page-course-with-cover.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * Title: Landing Page
```

Comment:
Is the title change intentional?

Reply:
It was requested [here](https://github.com/Automattic/sensei-theme/pull/49#issuecomment-1295159999)